src/main/scala/outwatch/util/Store.scala

```diff
@@ -1,33 +1,37 @@
 package outwatch.util
 
 import cats.effect.IO
-import outwatch.{Pipe, Sink}
+import monix.execution.Scheduler.Implicits.global
+import monix.execution.{Ack, Cancelable}
+import outwatch.{Handler, Pipe, Sink}
 import outwatch.dom._
 import outwatch.dom.helpers.STRef
-import rxscalajs.Observable
-import rxscalajs.subscription.Subscription
+
+import scala.concurrent.Future
+
+
 
 final case class Store[State, Action](initialState: State,
```

Comment:
`STRef` is only used in Store. Is it possible to remove it entirely?

Reply:
`STRef` is used in cases where a global variable with a referential transparent API is wanted (such as the store). It's possible to remove it, but without it the store would need to be passed as an argument to all components that use it.